src/nextjournal/clerk/sci_viewer.cljs

```diff
@@ -1139,7 +1136,20 @@ black")}]))}
 
    'doc-url doc-url
    'url-for url-for
-   'read-string read-string})
+   'read-string read-string
+
+   ;; clerk viewer API
+   'code code
+   'col col
+   'html html-render
```

Comment:
For symmetry,  I'd like to get `html` as exported from viewer ns, by setting the html-viewer's render-fn to `html-render` there. With that I'm running in all kind of endless recursion troubles, hard to un-🧶.

All calls to `(html x)` in sci-viewer ns seems to produce some unnecessary (?) indirection when going through the  `render-with-viewer` function.